lambdas/src/types/errors.ts

```diff
@@ -69,3 +69,19 @@ export class SessionValidationError extends BaseError {
         return `${this.code}: ${this?.details}`;
     }
 }
+
+export class AuthorizationCodeExpiredError extends BaseError {
+    constructor() {
+        super("Authorization code expired");
+        this.statusCode = 403;
+        this.code = 1027;
+    }
+}
+
+export class AccessCodeExpiredError extends BaseError {
```

Comment:
Also out of scope due to the above explanation

Reply:
Removed